### ISSUE TITLE
Play Sprint Plan 2019-12-16

### ISSUE THREAD

**Issue by renatocaval** [playframework/play-meta#144]
#### Meta TODOs

* [ ] Tweet (& blog?) about anything in the last sprint or this sprint?  E.g: noteworthy PRs or upcoming tasks.
  * [ ] play-grpc ready for GA releases
   
#### To discuss

* [x] @marcospereira update EOL information for Play and Lagom: https://github.com/lightbend/together-portal/blob/066ac8a015f4f4ab416e3388d1d74d3894a3f5ce/app/views/lightbendPlatform/supportedProjects.scala.html#L183
* [ ] @renatocaval gRPC samples need some love
    * [x] play-grpc with Play 2.8.0 and Lagom 1.6.0
    * [x] Re-add Play grpc samples in Travis build
    * [ ] Tweet about the samples and grpc when ready (See above)
* [x] @marcospereira MiMa checks unification
* [x] @renatocaval sbt-dynver all the things https://github.com/playframework/play-meta/issues/87
	- [x] [playframework/play-json](https://github.com/playframework/play-json) - https://github.com/playframework/play-json/pull/380
	- [x] [playframework/playframework](https://github.com/playframework/playframework) - https://github.com/playframework/playframework/pull/9929
	- [x] [playframework/play-slick](https://github.com/playframework/play-slick) - https://github.com/playframework/play-slick/pull/526
	- [x] [playframework/twirl](https://github.com/playframework/twirl) - https://github.com/playframework/twirl/pull/289
	- [x] [playframework/play-doc](https://github.com/playframework/play-doc) - https://github.com/playframework/play-doc/pull/78
	- [x] [playframework/scalatestplus-play](https://github.com/playframework/scalatestplus-play) - https://github.com/playframework/scalatestplus-play/pull/226
	- [x] [playframework/interplay](https://github.com/playframework/interplay) - https://github.com/playframework/interplay/pull/102
	- [x] [playframework/cachecontrol](https://github.com/playframework/cachecontrol) - https://github.com/playframework/cachecontrol/pull/106
	- [x] [playframework/play-file-watch](https://github.com/playframework/play-file-watch) - https://github.com/playframework/play-file-watch/pull/56

* [ ] @marcospereira Drop sbt 0.13
    * [x] Interplay https://github.com/playframework/interplay/pull/101
    * [x] Twirl https://github.com/playframework/twirl/pull/294
    * [x] Play https://github.com/playframework/playframework/pull/9936
    * [x] Lagom https://github.com/lagom/lagom/pull/2575
    * [x] ~~play-grpc~~ we don't have a Play plugin for grpc
    * [ ] lightbend-markdown
* [ ] @marcospereira Unblock Java 12+ in Scala Community Build
* [x] @ignasi35 (play and lagom but not libraries) Prefer Java 11 over Java 8 ( [play-meta#146](https://github.com/playframework/play-meta/issues/146))
    * [x] reorder jobs in CI (priority to Java11) - [play#9923](https://github.com/playframework/playframework/pull/9923) / [lagom#2564](https://github.com/lagom/lagom/pull/2564)
    * [ ] ~build/release using Java 11 JVM targeting java8~ _(won't do any change on `vegemite` and, instead, we'll wait for release from TravisCI)_
    * [ ] restore cross-linking javadocs to Akka - play changes will need to happen in `omnidoc` ([Omnidoc doesn't work on JDK11](https://github.com/playframework/omnidoc/issues/106)) / [lagom#2564](https://github.com/lagom/lagom/pull/2564)
* [ ] pull-request-validator
* [ ] @ignasi35 simplify docs (mostly changes in play and playfw.com, but lagom/play unification too)
    * [ ] extract play-docs and sbt-plugin-play-docs from playframework/playframework
    * [ ] play-doc to not use interplay
* [ ] reproduce minimised Coursier classpath ordering change https://github.com/sbt/sbt/issues/5282
* [x] Lagom: sbt 1.3.5 upgrade https://github.com/lagom/lagom/pull/2560
* [ ] @dwijnand Play dev mode maintenance
  * [x] drop sbt internals usage (Load)
  * [x] rm DocNoCache
  * [x] migrate off play-file-watch
* [x] @ignasi35 Play samples with telemetry [play-samples#76](https://github.com/playframework/play-samples/pull/76)

###### Metadata

* Dates: 2019-12-16 - 2019-12-20 + Jan 2-3/2020
* [Previous Sprint](https://github.com/playframework/play-meta/issues/138)
* Support Rota: Dale
* Person days: 15/15
